### PR TITLE
Update emacs from 27.1-1 to 27.2-2

### PR DESCRIPTION
**Pull request by mahori**
After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

### CODE REVIEW

**Comment by miccal** [comment]
The `binary`'s I removed are in separate `bin-*` folders for `arm` and `x86`.

File: Casks/emacs.rb [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-    regex(%r{href=.*?/Emacs-(\d+(?:\.\d+)*-\d+)-universal\.dmg}i)
+    regex(%r{href=.*?/Emacs-(\d+(?:\.\d+)*(?:-\d+)?)-universal\.dmg}i)
   end
 
   conflicts_with formula: "emacs"
 
   app "Emacs.app"
   binary "#{appdir}/Emacs.app/Contents/MacOS/Emacs", target: "emacs"
-  binary "#{appdir}/Emacs.app/Contents/MacOS/bin/ebrowse"
-  binary "#{appdir}/Emacs.app/Contents/MacOS/bin/emacsclient"
```
Comment (snarfed) [inline Casks/emacs.rb:20]: Hmm! I'm looking forward to Emacs 27.2, but I use emacsclient, and etags, and I'd hate to see them go away. Or will they still be included, and these line were just unnecessary?

**Comment by mahori** [comment]
[Emacs 27.2-2](https://emacsformacosx.com/download/emacs-builds/Emacs-27.2-2.changes) is released.
> Fix missing Emacs.app/Contents/MacOS/bin symlink

File: Casks/emacs.rb (hunk 1 above)
Comment (snarfed) [reply Casks/emacs.rb]: Ah, never mind, i see the answer in https://github.com/Homebrew/homebrew-cask/pull/102741#issuecomment-808690216

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**